source/content/global-cdn.md

```diff
@@ -68,10 +64,6 @@ No, we pre-configured the CDN so you don’t have to hassle with configuration,
 
 Hit rates are not currently available, but you can measure traffic for the Live environment. For details, see [Metrics in the Site Dashboard](/metrics).
 
-### Can I use other CDNs with the Pantheon Global CDN?
```

Comment:
I've combined the information from this section into the one in this PR because I think it's about the same thing. If I'm misreading, we can drop my commit and it's good to go. If it is the same topic, please review my changes/copy and merge

Reply:
:+1: